docs/posts/unified-stream-batch/index.qmd

```diff
@@ -0,0 +1,419 @@
+---
+title: "Stream-batch unification through Ibis lens"
```

Comment:
nitpicky, but should this just be "Stream-batch unification through Ibis"? not sure the "lens" at the end is doing much